### PR TITLE
api data applied

### PR DESCRIPTION
**Pull request by lenikasss**

### CODE REVIEW

File: src/components/introPage/IntroPage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      ? <div className={styles.introPage}>
+				<StatsTotal name={'Total Monthly Stats:'}
+					statsRings={[
+						{label: 'Github', percentage: 78},
+						{label: 'Jira', percentage: 42}
+					]}
+					labels={[{type: 'commits', value: 283}]}
+				/>
+
+				<Section>
```
Comment (lubomir1618) [inline src/components/introPage/IntroPage.tsx]: 🤔something wrong with indentation

File: src/components/introPage/IntroPage.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import StatsTotal from '../statstotal/StatsTotal';
+import { Section } from '../section/Section';
+import TeamWrapper from '../teamwrapper/TeamWrapper';
+import styles from './IntroPage.module.css';
+
+interface IProps {
+  fetchGit: () => void;
+  fetchJira: (path: string) => void;
+  gitData: any,
```
Comment (lubomir1618) [inline src/components/introPage/IntroPage.tsx]: comma?

Comment (lubomir1618) [reply src/components/introPage/IntroPage.tsx]: oh, I see 😄

Comment (lukasorgovan) [reply src/components/introPage/IntroPage.tsx]: nedziga vam obom ? :)

**Pull request merged**